javav2/README.rst

```diff
@@ -26,9 +26,9 @@ To build and run these examples, you need the following:
   
   After you set your AWS credentails in the credentials file located in the .aws folder, you can create a Service Client like this.

```

Comment:
"Service Client" -> "service client" (not a proper name).

Reply:
Fixed